zyte_spider_templates/spiders/ecommerce.py

```diff
@@ -32,6 +32,10 @@ class EcommerceCrawlStrategy(str, Enum):
     ignored. Use this when some subCategory links are misidentified by
     ML-extraction."""
 
+    none: str = "none"
+    """Treat input URLs as direct links to product detail pages, and
```

Comment:
Shall we also change the `url` input to accept newline-delimited URL values similar to the ongoing Article spider?

Reply:
Ahh sorry just saw the other PRs